docs/release_notes.md

```diff
@@ -1,4 +1,11 @@
 # Subscriptions API Release Notes
+<div class='post-date'>04 November 2020 - Invoice fields</div> 
+Merchants can add [invoice](invoice) for susbscription payments without user's information.
+Fields not mandatory: consumer_name, consumer_phone_number, consumer_address_lines.
+If the fields are not filled, they are not displayed in PDF.
+Can be negative: TotalVatAmount, TotalPriceIncludingVat, PricePerUnit, PriceDiscount.
+10 decimals allowed for: Quantity, PricePerUnit.
```

Comment:
"10 places after the separator" ? You don't allow 10 different numbers here, I guess

Reply:
Decimal also refers specifically to the digits after the decimal separator, such as in "3.14 is the approximation of π to two decimals".
I think all of the error messages in RecurringPayments use this kind of naming "no more than 10 decimals" and etc. Is it too ambiguous?

Reply:
In the new commit I made it as easily understandable as possible.